src/base/bittorrent/torrentinfo.cpp

```diff
@@ -63,7 +63,7 @@ TorrentInfo TorrentInfo::loadFromFile(const QString &path, QString &error)
 {
     error.clear();
     libt::error_code ec;
-    TorrentInfo info(new libt::torrent_info(Utils::String::toStdString(Utils::Fs::toNativePath(path)), ec));
+    TorrentInfo info(NativePtr(new libt::torrent_info(Utils::String::toStdString(Utils::Fs::toNativePath(path)), ec)));
```

Comment:
Utils::String::toStdString produces utf8 string. We did not use a wide strings to do this (at least the last few years). We, apparently, simply forgot to remove TORRENT_USE_WPATH definition earlier.